base/src/main/java/io/spine/base/Identifier.java

```diff
@@ -246,17 +263,20 @@ public static String newUuid() {
     /**
      * Converts the passed ID value into the string representation.
      *
-     * @param id  the value to convert
-     * @param <I> the type of the ID
+     * @param id
+     *         the value to convert
+     * @param <I>
+     *         the type of the ID
      * @return <ul>
-     * <li>for classes implementing {@link Message} &mdash; a Json form;
-     * <li>for {@code String}, {@code Long}, {@code Integer} &mdash;
-     * the result of {@link Object#toString()};
-     * <li>for {@code null} ID &mdash; the {@link #NULL_ID};
-     * <li>if the result is empty or blank string &mdash; the {@link #EMPTY_ID}.
-     * </ul>
-     * @throws IllegalArgumentException if the passed type isn't one of the above or
-     *                                  the passed {@link Message} instance has no fields
+     *         <li>for classes implementing {@link Message} &mdash; a Json form;
```

Comment:
Let's get rid of `&mdash;` and simply use `-` (minus) instead. This would improve readability of the raw Javadoc. 

Given that the compiled Javadoc is not read as frequently as its raw form in the real world, this is a good thing.